GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -458,7 +458,7 @@ protected override void OnFormClosed(FormClosedEventArgs e)
             {
                 // Do not remember commit message of fixup or squash commits, since they have
                 // a special meaning, and can be dangerous if used inappropriately.
-                if (CommitKind == CommitKind.Normal)
+                if (CommitKind is CommitKind.Normal or CommitKind.Amend)
```

Comment:
Parens are required here
```suggestion
                if (CommitKind is (CommitKind.Normal or CommitKind.Amend))
```